app/scripts/api/articlesService.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = [
+  '$http',
+  '$window',
+  'API_URL',
+
+  function articles($http, $window, API_URL) {
+
+      function getArticles() {
+          return $http.get(API_URL+'articles/');
+      }
+
+      function createNewArticle(author) {
+          return $http.post(API_URL + 'articles/', {
+                    authors  : [ parseInt(author)],
```

Comment:
You're missing `radix` parameter. Omiting radix can lead to unexpected consequences, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt